### ISSUE TITLE
Add wildcard support for server browser

### ISSUE THREAD

**Issue by micwoj92** [mastercomfig/team-comtress-2#40]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I cant find map.

**Describe the solution you'd like**
For example searching *badlands* would list cp_badlands, kotf_badlands etc.

**Describe alternatives you've considered**
none

**Additional context**
My preferred wildcard sign is *

**Comment by tom288** [comment]
The server browser search is very slow & I imagine this might make it much worse if the string matching is the slowest part
I agree this would be very useful

**Comment by micwoj92** [comment]
> The server browser search is very slow

I do not agree. Refreshing it is indeed slow, but once you have all servers searching it fast.

**Comment by maximsmol** [comment]
The server browser is slow because it has to fetch the list, the search itself is basically instant
